actions/actionsRouter.js

```diff
@@ -0,0 +1,65 @@
+const express = require("express");
+const Actions = require("../data/helpers/actionModel");
+const router = express.Router();
+
+//get
+router.get("/", (req, res) => {
+  Actions.get()
+    .then((result) => {
+      if (result) {
+        res.status(200).json(result);
+      } else {
+        res.status(404).json({ message: "actions not found" });
+      }
+    })
+    .catch((err) => {
+      res.status(500).json({ message: "error, can't get anything" });
+    });
+});
+
+//post
+router.post("/:id", (req, res) => {
+  Actions.insert(req.body)
+    .then((result) => {
```

Comment:
Also, this one was tricky, but because an action requires `project_id`, we want to put the endpoint for adding an action in the projectsRouter ('/:id/actions').  That's because the id at this endpoint is the action id, and we want to access the project id that this action will be added to